### PR TITLE
fix(route): 修正南京大学本科生院教学信息网信息获取

### PR DESCRIPTION
**Pull request by cqjjjzr**
## 该 PR 相关 Issue / Involved issue

（无）

## 完整路由地址 / Example for the proposed route(s)

```routes
/nju/jw/ggtz
/nju/jw/jxdt
```

## 说明 / Note

将 HTML 解释改为利用 WebPlus 的 REST 接口。通过自定义 `returnInfos` 可以实现 返回格式的自定义，特别是可以获取时间信息。增大每次返回条目数。

### CODE REVIEW

**Comment by cqjjjzr** [comment]
CC @ret-1

**Review by TonyRL — COMMENTED** [review]
Please update maintainer.js as well.

**Review by TonyRL — CHANGES_REQUESTED** [review]
This API endpoint is returning outdated feeds.
Please cross-check with https://jw.nju.edu.cn/ggtz/list.htm and https://jw.nju.edu.cn/_s414/24774/list.psp

**Comment by cqjjjzr** [comment]
I don't think so. As I mentioned before, you should pull the feed with `sorted=false` since the returned 

However, I noticed the search order (in the search criteria) and time parsing are wrong. Pushed to fix.

File: lib/v2/nju/jw.js [hunk 1]
```diff
@@ -1,45 +1,57 @@
...
-    const $ = cheerio.load(data);
-    const list = $('li.news');
-
     ctx.state.data = {
         title: `本科生院-${type_dict[type][1]}`,
-        link: type_dict[type][0],
-        item: list
-            .map((index, item) => {
-                item = $(item);
+        link: `https://jw.nju.edu.cn/${type_dict[type][0]}/list.htm`,
```
Comment (TonyRL) [inline lib/v2/nju/jw.js:40]: https://jw.nju.edu.cn/26263/list.htm will not work, it should be https://jw.nju.edu.cn/ggtz/list.htm

**Pull request merged**